crates/red_knot/src/python_version.rs

```diff
@@ -1,68 +0,0 @@
-/// Enumeration of all supported Python versions
-///
-/// TODO: unify with the `PythonVersion` enum in the linter/formatter crates?
-#[derive(Copy, Clone, Hash, Debug, PartialEq, Eq, PartialOrd, Ord, Default, clap::ValueEnum)]
```

Comment:
I think I'd like to keep this type because I don't want the parser to depend on clap.